sdk/storage/azure-storage-files-datalake/test/datalake_sas_test.cpp

```diff
@@ -51,10 +52,10 @@ namespace Azure { namespace Storage { namespace Test {
     directory1Client0.Create();
     directory2Client0.Create();
 
-    auto serviceUri = serviceClient0.GetDfsUri();
-    auto filesystemUri = filesystemClient0.GetDfsUri();
-    auto directory1Uri = directory1Client0.GetDfsUri();
-    auto directory2Uri = directory2Client0.GetDfsUri();
+    auto serviceUri = Files::DataLake::Details::GetDfsUriFromUri(serviceClient0.GetUri());
```

Comment:
We should be consistent and call these APIs `GetUrl` not `GetUri`, similar to:
https://github.com/Azure/azure-sdk-for-cpp/blob/e0e04cfee04988dad0a2bcbbbd1e40fc04da436a/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_container_client.hpp#L122

For example, change this (and everywhere else `Uri` shows up):
https://github.com/Azure/azure-sdk-for-cpp/blob/e49ceabf85b2250c08241c27a52bbfaddbef1b85/sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_service_client.hpp#L67

Reply:
It should be `GetUrl`